recipe/bld.bat

```diff
@@ -5,6 +5,7 @@ set MENU_DIR=%PREFIX%\Menu
 IF NOT EXIST (%MENU_DIR%) mkdir %MENU_DIR%
 
 copy %SRC_DIR%\img_src\spyder.ico %MENU_DIR%\
+copy %SRC_DIR%\img_src\spyder_reset.ico %MENU_DIR%\
```

Comment:
I don't think this line is needed as you aren't using the reset icon, but the spyder.ico file below.   Or maybe in the future you will update the spyder_reset.ico and this makes that PR easier?

Reply:
> Or maybe in the future you will update the spyder_reset.ico and this makes that PR easier?
Exactly! That's the idea :+1: